README.md

```diff
@@ -22,7 +20,55 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+    $ psql
+
+```sql
+CREATE DATABASE storage;
```

Comment:
why do you need a local storage at all? 
ok, even you need it, why should I care about storage as a user? What if I specify sqlite here?